src/vaults/Escrow.sol

```diff
@@ -77,12 +93,9 @@ contract PoolEscrow is Escrow, Recoverable, IPoolEscrow {
 
     /// @inheritdoc IPoolEscrow
     function deposit(bytes16 scId, address asset, uint256 tokenId, uint256 value) external auth {
-        _deposit(scId, asset, tokenId, value, true);
-    }
+        holding[scId][asset][tokenId].total += value.toUint128();
```

Comment:
Q: Could you explain to me quickly why we could drop the previous sufficiency checks? Is it because we moved the check to `SyncRequests._maxDeposit`?

Reply:
What do you mean? My assumption here was that we do not need the sufficiency as we can't control the inflow. I.e. code using `noteDeposit` and trusts that higher level transfers. So I thought I make it slimmer